### PR TITLE
Update checklist.bs

### PR DESCRIPTION
**Pull request by ArunavC**

### CODE REVIEW

File: checklist.bs [hunk 1]
```diff
@@ -288,210 +288,209 @@ syntax specified in [[DXP#api-action-get-request]].
 The test target host system should respond with a 404 Not Found and a JSON body containing the error
 code `NoSuchFootprint`, as specified in [[DXP#api-error-responses]].
 
+## Test Case 012: Receive Notification of PCF Update ## {#tc012}
+
+Advisement: This test case will become mandatory test case with the release of version 2.2 of the Technical Specifications.
```
Comment (bhadley) [inline checklist.bs:293]: Revise / remove the Advisement, as it is not logically relevant anymore (v2.2 is released). Would revise to make reference somewhere that this test case is only mandatory if the tester is testing conformance to V2.2 or higher of the specifications. This applies to all advisements similar to this one...

File: checklist.bs [hunk 2]
```diff
@@ -288,210 +288,209 @@ syntax specified in [[DXP#api-action-get-request]].
...
+### Expected Response ### {#tc012-response}
+
+The test target host system must respond with 200 OK and an empty body.
+
+If the test target host system calls the GetFootprint action with the `pfId` included in the
+notification, the corresponding PCF must be returned.
+
+## Test Case 013: Notify of PCF Update ## {#tc013}
+
+Advisement: This test case will become mandatory test case with the release of version 2.2 of the Technical Specifications.
```
Comment (bhadley) [inline checklist.bs:311]: Same comment as above

File: checklist.bs [hunk 3]
```diff
@@ -288,210 +288,209 @@ syntax specified in [[DXP#api-action-get-request]].
...
+### Expected Response ### {#tc013-response}
+
+If the testing party has implemented the Events functionality, it should respond with 200 OK and an empty body.
+
+Otherwise, it should respond with 400 Bad Request and a JSON body containing the error response
+`NotImplemented`, as specified in [[DXP#api-error-responses]].
+
+## Test Case 014: Asynchronous PCF Request ## {#tc014}
+
+Advisement: This test case will become mandatory test case with the release of version 2.2 of the Technical Specifications.
```
Comment (bhadley) [inline checklist.bs:336]: Same as above

File: checklist.bs [hunk 4]
```diff
@@ -288,210 +288,209 @@ syntax specified in [[DXP#api-action-get-request]].
...
+A POST request must be sent to the test target host system's `/2/events` endpoint with the syntax
+specified in [[DXP#api-action-events-case-2-request]].
+
+### Expected Response ### {#tc014-response}
+
+The test target host system must respond with 200 OK.
+
+## Test Case 015: Respond to Asynchronous PCF Request ## {#tc015}
+
+Advisement: This test case will become mandatory test case with the release of version 2.2 of the Technical Specifications.
```
Comment (bhadley) [inline checklist.bs:351]: Same

File: checklist.bs
Comment (bhadley) [inline checklist.bs:1]: Currently it is not clear to users which version of the tech specs this checklist applies to (hence the question from Siemens). We should clarify this in the Introduction. Would just add a few sentences to the Introduction paragraph, something like "This document is intended to be used as a checklist to validate conformance to any version of the Technical Specifications. The Required test cases indicate mandatory functionality per the latest version of the Technical Specification. When functionality is optional in the version of the specification an organization is testing conformance to, that test case is considered optional." 

And then maybe adjust the definition in the Terminology of Required and Optional

Required Test Case
Test case relating to MANDATORY functionality per the latest version of the Technical Specifications.

Optional Test Case
Test cases relating to OPTIONAL functionality per the latest version of the Technical Specifications.

**Review by bhadley — CHANGES_REQUESTED** [review]
Looking good, just added a few comments for revision! Happy to re-review once you revise!

File: checklist.bs
Comment (ArunavC) [reply checklist.bs:1]: done

File: checklist.bs (hunk 1 above)
Comment (ArunavC) [reply checklist.bs]: done

File: checklist.bs (hunk 2 above)
Comment (ArunavC) [reply checklist.bs]: done

File: checklist.bs (hunk 3 above)
Comment (ArunavC) [reply checklist.bs]: done

File: checklist.bs (hunk 4 above)
Comment (ArunavC) [reply checklist.bs]: done

**Review by bhadley — APPROVED** [review]
LGTM!

**Pull request merged**